servidor.py

```diff
@@ -1,9 +1,8 @@
-from flask import Flask, request, jsonify, json
+from flask import Flask, request, session, jsonify, render_template
 from servicios.autenticacion import autenticacion
-from flask import render_template
 
 app = Flask(__name__)
-
+app.secret_key = "unaClaveMuySeguraQueDeberiaSerAlmacenadaEn .env"
```

Comment:
secret_key is what flask uses to "encrypt" the cookies of the session. In production this should be stored in a separated config file, preferably a .env (dotenv) file.